Update README.md (#21272)
DataDog/integrations-core: teleport/README.md

### Service Checks

The Teleport integration does not include any service checks. 

> This parameter 'enable_health_service_check' enabled by default will send a service check named `<NAMESPACE>.openmetrics.health` which reports the health of the `openmetrics_endpoint`

Reviewer: ```suggestion
The parameter 'enable_health_service_check', which is enabled by default, sends a service check named `<NAMESPACE>.openmetrics.health`, which reports the health of the `openmetrics_endpoint`.
```
